ibm/mas_devops/roles/suite_app_config/tasks/main.yml

```diff
@@ -97,6 +97,18 @@
       - app_cr_result.resources | json_query('[*].status.conditions[?type==`Ready`][].status') | select ('match','True') | list | length == 1
     fail_msg: "Workspace {{ mas_workspace_id }} created but application {{ mas_app_id }} is not ready"
 
+# Manage: RedHat AMQ stream certificate setup
+- name: "Debug mas_manage_kafka_provider"
+  debug:
+    msg:
+      - "mas_manage_kafka_provider ....... {{ mas_manage_kafka_provider }}"
+
+- name: "Manage specific : Set redhat AMQ stream certificate"
+  when:
+    - mas_manage_kafka_provider is defined
+    - mas_manage_kafka_provider == "redhat"
+  include_tasks: "tasks/manage/setup-redhat-amq-certificate.yml"
```

Comment:
This will always run regardless of if it is manage or not, or even if you have manage installed. If needs a check to make sure this only runs if the app is manage

Reply:
Added condition to check app is manage.